layouts/partials/landingpage/most-popular.html

```diff
@@ -15,17 +15,17 @@ <h4 class="card-title">String Function Calls</h4>
 		<div class="card-body">
 			<h4 class="card-title">Development Best Practices</h4>
 			<br>
-			<a href="/howto/general/dev-best-practices" class="btn btn-outline-secondary doc-link">Go to doc</a>
+			<a href="/howto/general/dev-best-practices/" class="btn btn-outline-secondary doc-link">Go to doc</a>
 		</div>
 		</div>
 	</div>
 	<div class="col-12 col-sm-6 col-xl-4 image-card-col">
 		<div class="card image-card">
-		<img src="/images/most-popular-system-requirements.png" class="card-img-top" alt="...">
+		<img src="/images/most-popular-expressions.png" class="card-img-top" alt="...">
 		<div class="card-body">
-			<h4 class="card-title">System Requirements</h4>
+			<h4 class="card-title">Expressions</h4>
 			<br>
-			<a href="/refguide/system-requirements" class="btn btn-outline-secondary doc-link">Go to doc</a>
+			<a href="/refguide/expressions/" class="btn btn-outline-secondary doc-link">Go to doc</a>
```

Comment:
Is there a way to make the cards a fixed height? It looks a bit ugly when I don't use the whole screen width.
(LOW PRIORITY!)
![image](https://user-images.githubusercontent.com/35492184/157215683-ede26d73-6f24-4be0-8197-cf8d423e915f.png)

Reply:
I will add this to the to do list and see.